lascli/actions/workflows.py

```diff
@@ -0,0 +1,241 @@
+        
+    return docker_secret.get('secretId'), cradl_secret['secretId']
+
+
+@wrap_output(start_msg='Creating remote component and field config asset ...', end_msg='Done')
+@capture_return(dest=created_ids['assets'])
+def create_assets(las_client: Client, model_id: str, create_tag: str):
+    component = las_client.create_asset(
+        content=requests.get(COMPONENT_ASSEST_URI).content,
+        name='Manual validation component',
+        description=create_tag
+    )
+    
+    field_config = las_client.create_asset(
+        content=json.dumps(create_field_config(las_client, model_id)).encode('utf-8'),
+        name='Field configuration',
+        description=create_tag
+    )
+
+    return component['assetId'], field_config['assetId']
+
+
+@wrap_output(start_msg='Creating dataset ...', end_msg='Done')
+@capture_return(dest=created_ids['datasets'])
+def create_dataset(las_client: Client, name: str, create_tag: str):
+    dataset = las_client.create_dataset(name=name, description=f'Dataset for {name} model. {create_tag}')
+    return [dataset['datasetId']]
+    
+
+@wrap_output(start_msg='Creating transitions ...', end_msg='Done')
+@capture_return(dest=created_ids['transitions'])
+def create_transitions(
+    las_client: Client, cradl_secret_id: str, parameters: dict,
+    create_tag: str, created_ids: dict, docker_secret_id: str=None
+):
+    docker_auth_details = {'secretId': parameters['secret_id']} if parameters.get('secret_id') else {}
+
+    common_params = {
+        'environmentSecrets': [cradl_secret_id],
+        'environment': {
+            'MODEL_ID': parameters['model_id'],
+            'FIELD_CONFIG_ASSET_ID': parameters['field_config_asset_id'],
+            'DATASET_ID': parameters['dataset_id']
+        },
+        **docker_auth_details
+    }
+    
+    preprocess = las_client.create_transition(
+        transition_type='docker',
+        parameters={'imageUrl': parameters['preprocess_image'], **common_params},
+        name=f'Preprocess transition for workflow {parameters["name"]}',
+        description=create_tag
+    )
+    
+    postprocess = las_client.create_transition(
+        transition_type='docker',
+        parameters={'imageUrl': parameters['postprocess_image'], **common_params},
+        name=f'Postprocess transition for workflow {parameters["name"]}',
+        description=create_tag
+    )
+
+    manual = las_client.create_transition('manual', parameters={
+        'assets': {
+            'jsRemoteComponent': parameters['remote_component_asset_id'],
+            'fieldConfig': parameters['field_config_asset_id']
+        }
+    }, name=parameters['name'], description=create_tag)
+
+
+    return preprocess['transitionId'], postprocess['transitionId'], manual['transitionId']
+
+
+def create_default_workflow(las_client: Client, name: str, **optional_args):
+    timestamp = datetime.now()
+    create_tag = f'Created by CLI at {timestamp.isoformat()} (tag:{str(uuid.uuid4())})'
```

Comment:
Subjective, but I like this compact version better

```suggestion
    create_tag = f'Created by CLI at {timestamp.isoformat()} (tag:{uuid.uuid4().hex})'
```